app/assets/sass/modules/_button-reset.scss

```diff
@@ -15,6 +15,20 @@
   &:hover {
     color: $govuk-link-hover-colour;
   }
+
+  &:focus {
```

Comment:
Could this be using the mixin?

The box-shadow's look like a pre-release version of the focus.

https://design-system.service.gov.uk/get-started/focus-states/

Reply:
Ah nice one, will update